src/analysis/hands.cc

```diff
@@ -836,17 +836,22 @@ int isThreeKans(const GameState& state, int player,
 
 int isAllPons(const GameState& state, int player,
```

Comment:
I feel like there is something to be said of the return 0 short circuits here, but we're dealing with so few nodes max this is much simpler to read

Reply:
Optimization later.